app/main.py

```diff
@@ -21,15 +23,19 @@ def serve_cars(self, cars: list[Car]) -> float:
             self.wash_single_car(i)
         return income
 
-    def calculate_washing_price(self, car: Car):
-        return round(car.comfort_class * (self.clean_power - car.clean_mark) * (
-                    self.average_rating / self.distance_from_city_center), 1)
+    def calculate_washing_price(self, car: Car) -> float:
+        one = self.clean_power - car.clean_mark
```

Comment:
It would be nice to use more meaningful variables name. For example, "clean_level_difference" instead of "one", or "rating_dist _ratio" instead of "two"